FAQ.md

```diff
@@ -0,0 +1,52 @@
+# FAQ - Frequently Asked Questions
+
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
```

Comment:
😅

Reply:
😶‍🌫️ this is auto-generated, can be removed if proven to be cumbersome. Happy to keep maintaining the TOCs :)